<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingform</class>
 <widget class="QWidget" name="settingform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="dock_line2">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>34</y>
     <width>10</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="dock_line">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>34</y>
     <width>10</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="savejson_button">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>30</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkequip_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>28</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>检测设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadjson_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>181</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_layout">
    <item>
     <widget class="QLineEdit" name="name_lineedit">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>雨夜</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="sp1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="ChatGPTmodel_combobox">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>gpt-3.5-turbo</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="sp2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="userprompt_lineedit">
      <property name="placeholderText">
       <string>./prompt.txt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="jsonfile_lineedit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>./userdata.json</string>
   </property>
  </widget>
  <widget class="QLabel" name="tips_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>配置文件</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_browser">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>100</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="dock_line3">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>34</y>
     <width>10</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="newjson_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="Line" name="dock_line4">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>34</y>
     <width>10</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cleartemp_button">
   <property name="geometry">
    <rect>
     <x>849</x>
     <y>28</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空缓存</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>96</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>虚拟人名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_name_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>129</y>
     <width>240</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIY虚拟人怎么称呼呢？</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ChatGPTmodel_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>233</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GPT模型</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_ChatGPTmodel_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>266</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPT-4效果好，您的API不支持时，还是会使用GPT3.5</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="userprompt_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>364</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>角色设定</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_userprompt_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>395</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>一个txt文件，说明你的虚拟人设定</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tip_upkey_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>304</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>看声音模型，女生填0，男生填-12。出现撕裂可以微调。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="device_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>202</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>推理设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="method_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>362</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>推理模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="upkey_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>283</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>声音变调</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_voicemodel_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>151</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.pth和.index两个文件，缺一不可</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>181</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_layout2">
    <item>
     <widget class="QLineEdit" name="voicemodel_pth_lineedit">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>./AI**.pth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="voicemodel_index_lineedit">
      <property name="placeholderText">
       <string>./AI**.index</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="sp4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="device_combobox">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>GPU</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="sp6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="upkey_lineedit">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>-12~+12</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="sp5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="method_combobox">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>crepe</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="voicemodel_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>97</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>声音模型</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_method_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>crepe:使用GPU，有显卡首选
pm:使用CPU，高速低质
harvest:使用CPU，低速高质</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tip_device_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>231</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>如果有显卡，首选GPU，不支持再选CPU</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="form_line">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>16</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/xiaomeng.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_label2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 130);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="form_line2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>16</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="dock_line5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>34</y>
     <width>10</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="setplus_button">
   <property name="geometry">
    <rect>
     <x>968</x>
     <y>28</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>高级设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_json_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>38</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_userprompt_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>369</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_pth_button">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>99</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_index_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>125</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <zorder>back_label</zorder>
  <zorder>back_label2</zorder>
  <zorder>dock_line2</zorder>
  <zorder>dock_line</zorder>
  <zorder>savejson_button</zorder>
  <zorder>checkequip_button</zorder>
  <zorder>loadjson_button</zorder>
  <zorder>verticalLayoutWidget_4</zorder>
  <zorder>jsonfile_lineedit</zorder>
  <zorder>tips_label</zorder>
  <zorder>output_browser</zorder>
  <zorder>dock_line3</zorder>
  <zorder>newjson_button</zorder>
  <zorder>dock_line4</zorder>
  <zorder>cleartemp_button</zorder>
  <zorder>name_label</zorder>
  <zorder>tip_name_label</zorder>
  <zorder>ChatGPTmodel_label</zorder>
  <zorder>tip_ChatGPTmodel_label</zorder>
  <zorder>userprompt_label</zorder>
  <zorder>tip_userprompt_label</zorder>
  <zorder>tip_upkey_label</zorder>
  <zorder>device_label</zorder>
  <zorder>method_label</zorder>
  <zorder>upkey_label</zorder>
  <zorder>tip_voicemodel_label</zorder>
  <zorder>verticalLayoutWidget_5</zorder>
  <zorder>voicemodel_label</zorder>
  <zorder>tip_method_label</zorder>
  <zorder>tip_device_label</zorder>
  <zorder>form_line</zorder>
  <zorder>form_line2</zorder>
  <zorder>dock_line5</zorder>
  <zorder>setplus_button</zorder>
  <zorder>select_json_button</zorder>
  <zorder>select_userprompt_button</zorder>
  <zorder>select_pth_button</zorder>
  <zorder>select_index_button</zorder>
 </widget>
 <resources>
  <include location="NightRain.qrc"/>
 </resources>
 <connections/>
</ui>
